commit 241ca51e3dcde6976966ade50cee4746a348a9a9
Author: Matthias Reichl <hias@horus.com>
Date:   Fri Feb 2 20:30:43 2018 +0100

    ASoC: justboom-dac: fix S24_LE format
    
    Remove set_bclk_ratio call so 24-bit data is transmitted in
    24 bclk cycles.
    
    Also remove hw_params as it's no longer needed.
    
    Signed-off-by: Matthias Reichl <hias@horus.com>

diff --git a/sound/soc/bcm/justboom-dac.c b/sound/soc/bcm/justboom-dac.c
index 9bab6cf063d3..217626cdad6c 100644
--- a/sound/soc/bcm/justboom-dac.c
+++ b/sound/soc/bcm/justboom-dac.c
@@ -49,17 +49,6 @@ static int snd_rpi_justboom_dac_init(struct snd_soc_pcm_runtime *rtd)
 	return 0;
 }
 
-static int snd_rpi_justboom_dac_hw_params(struct snd_pcm_substream *substream,
-				       struct snd_pcm_hw_params *params)
-{
-	struct snd_soc_pcm_runtime *rtd = substream->private_data;
-	struct snd_soc_dai *cpu_dai = rtd->cpu_dai;
-	/*return snd_soc_dai_set_bclk_ratio(cpu_dai, 64);*/
-	unsigned int sample_bits =
-		snd_pcm_format_physical_width(params_format(params));
-	return snd_soc_dai_set_bclk_ratio(cpu_dai, sample_bits * 2);
-}
-
 static int snd_rpi_justboom_dac_startup(struct snd_pcm_substream *substream) {
 	struct snd_soc_pcm_runtime *rtd = substream->private_data;
 	struct snd_soc_codec *codec = rtd->codec;
@@ -75,7 +64,6 @@ static void snd_rpi_justboom_dac_shutdown(struct snd_pcm_substream *substream) {
 
 /* machine stream operations */
 static struct snd_soc_ops snd_rpi_justboom_dac_ops = {
-	.hw_params = snd_rpi_justboom_dac_hw_params,
 	.startup = snd_rpi_justboom_dac_startup,
 	.shutdown = snd_rpi_justboom_dac_shutdown,
 };
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1194_241ca51e3dcde6976966ade50cee4746a348a9a9.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1194_241ca51e3dcde6976966ade50cee4746a348a9a9.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
